cs-vmware/src/main/java/io/cloudslang/content/vmware/actions/deployment/DeployOvfTemplateAction.java

```diff
@@ -0,0 +1,156 @@
+                                              @Param(value = Inputs.CLUSTER_NAME) String clusterName,
+                                              @Param(value = Inputs.RESOURCE_POOL) String resourcePool,
+                                              @Param(value = Inputs.VM_FOLDER) String vmFolder,
+                                              @Param(value = Inputs.DISK_PROVISIONING) String diskProvisioning,
+                                              @Param(value = Inputs.IP_PROTOCOL) String ipProtocol,
+                                              @Param(value = Inputs.IP_ALLOC_SCHEME) String ipAllocScheme,
+                                              @Param(value = Inputs.LOCALE_LANG) String localeLang,
+                                              @Param(value = Inputs.LOCALE_COUNTRY) String localeCountry,
+                                              @Param(value = Inputs.OVF_NETWORK_JS) String ovfNetworkJS,
+                                              @Param(value = Inputs.NET_PORT_GROUP_JS) String netPortGroupJS,
+                                              @Param(value = Inputs.OVF_PROP_KEY_JS) String ovfPropKeyJS,
+                                              @Param(value = Inputs.OVF_PROP_VALUE_JS) String ovfPropValueJS,
+                                              @Param(value = Inputs.PARALLEL) String parallel) {
+        try {
+            Locale locale = InputUtils.getLocale(localeLang, localeCountry);
+
+            HttpInputs httpInputs = new HttpInputs.HttpInputsBuilder()
+                    .withHost(host)
+                    .withPort(port)
+                    .withProtocol(protocol)
+                    .withUsername(username)
+                    .withPassword(password)
+                    .withTrustEveryone(trustEveryone)
+                    .build();
+
+            VmInputs vmInputs = new VmInputs.VmInputsBuilder()
+                    .withHostname(hostname)
+                    .withVirtualMachineName(name)
+                    .withDataCenterName(datacenter)
+                    .withDataStore(dataStore)
+                    .withCloneDataStore(dataStore)
+                    .withFolderName(vmFolder)
+                    .withLocale(locale)
+                    .withClusterName(clusterName)
+                    .withResourcePool(resourcePool)
+                    .withIpProtocol(ipProtocol)
+                    .withIpAllocScheme(ipAllocScheme)
+                    .withDiskProvisioning(diskProvisioning)
+                    .build();
+
+            Map<String, String> ovfNetworkMappings = OvfUtils.getOvfMappings(ovfNetworkJS, netPortGroupJS);
+            Map<String, String> ovfPropertyMappings = OvfUtils.getOvfMappings(ovfPropKeyJS, ovfPropValueJS);
+
+            new DeployOvfTemplateService(InputUtils.getBooleanInput(parallel, true)).deployOvfTemplate(httpInputs, vmInputs, path, ovfNetworkMappings, ovfPropertyMappings);
```

Comment:
Because VmInputs was created to identify a Virtual Machine on vcenter host and contains mainly virtual machine specific inputs. These three inputs are only used by this Deploy Ovf Template operation since they are operation specific and I don't think they will be used for other operations.